docs/modules/ssh_key_list.md

```diff
@@ -0,0 +1,81 @@
+# ssh_key_list
```

Comment:
Could you re-run `make gendocs` to get the docs validation workflow passing again? The whitespace issue will be fixed once we bump and update for `ansible-specdoc`.